.circleci/config.yml

```diff
@@ -462,8 +445,8 @@ jobs:
           password: $DOCKER_EAPD_PW
     steps:
       - checkout
-      - run: yarn add glob
-      - run: yarn add js-yaml
+      - run: npm i glob
```

Comment:
This seems backwards to me, I'm assuming this is an intentional choice to go back to npm i rather than using yarn?

Reply:
I think yarn was giving me some trouble but I can switch it back

Reply:
That's not necessary, was just curious